src/core/ddsc/tests/entity_contains.c

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright(c) 2006 to 2018 ADLINK Technology Limited and others
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0, or the Eclipse Distribution License
+ * v. 1.0 which is available at
+ * http://www.eclipse.org/org/documents/edl-v10.php.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR BSD-3-Clause
+ */
+#include "ddsc/dds.h"
+#include "CUnit/Test.h"
+#include "CUnit/Theory.h"
```

Comment:
No need to include CUnit/Test.h directly. It's included automatically when CUnit/Theory.h is included.